playbooks/roles/ses-rook/tasks/setup_for_openstack.yml

```diff
@@ -0,0 +1,109 @@
+---
+# 
+# (c) Copyright 2019 SUSE Linux GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+# This is reused from automation/scripts/jenkins/ses/ansible/roles/ses/tasks/openstack_config.yml
+
+- name: Create openstack pool(s)
+  shell: |
+    set -o pipefail
+    kubectl -n rook-ceph exec $(kubectl -n rook-ceph get pod -l 'app=rook-ceph-tools' -o jsonpath='{.items[0].metadata.name}') -- \
+    ceph osd pool create {{ item.name }} {{ item.pg_num }}
+  with_items: "{{ ses_openstack_pools | unique }}"
+  changed_when: false
+  failed_when: false
+
+- name: Find out ceph-tools pod
+  command: "kubectl -n rook-ceph get pod -l 'app=rook-ceph-tools' -o jsonpath='{.items[0].metadata.name}'"
+  register: _rook_ceph_tools_pod
+
+- name: Remember ceph-tools pod name
+  set_fact:
+    ceph_tools_pod: "{{ _rook_ceph_tools_pod.stdout }}"
+
+- name: Create openstack key(s)
+  shell: |
+    set -o pipefail
+    kubectl -n rook-ceph exec {{ ceph_tools_pod }} -- \
+    bash -c "ceph-authtool -C /etc/ceph/ceph.{{ item.name }}.keyring --name {{ item.name }} --add-key \$(ceph-authtool --gen-print-key) --cap mon \"{{ item.mon_cap|default('') }}\" --cap osd \"{{ item.osd_cap|default('') }}\""
+  args:
+    creates: "/etc/ceph/ceph.{{ item.name }}.keyring"
+  with_items: "{{ ses_openstack_keys }}"
+  changed_when: false
+  no_log: true
+
+- name: Check if openstack key(s) already exist(s)
+  command: "kubectl -n rook-ceph exec {{ ceph_tools_pod }} -- ceph auth get {{ item.name }}"
+  changed_when: false
+  failed_when: false
+  with_items: "{{ ses_openstack_keys }}"
+  register: openstack_key_exist
+  no_log: true
+
+- name: Add openstack key(s) to ceph
+  command: "kubectl -n rook-ceph exec {{ ceph_tools_pod }} -- ceph auth import -i /etc/ceph/ceph.{{ item.0.name }}.keyring"
+  changed_when: false
+  with_together:
+    - "{{ ses_openstack_keys }}"
+    - "{{ openstack_key_exist.results }}"
+  when: item.1.rc != 0
+  no_log: true
+
+- name: Get ses conf options
+  shell:
+    set -o pipefail
+    kubectl -n rook-ceph exec {{ ceph_tools_pod }} -- \
+    grep 'fsid\|mon_host\|mon_initial\|network' /etc/ceph/ceph.conf | awk '{ print $3}'
+  register: conf_options
```

Comment:
Hm, this does not seem to be used, I should remove it later

Reply:
Update to myself: it was used for `ceph_conf:` values, let's wait if we need it...

Reply:
Hm, originally we were saving all these values from ceph.conf into our `ses_config.yml`. However I do not see any place where they'd be used, is it completely useless part? @Itxaka ?

Reply:
This just gets the mons? Im not sure we need that anymore, airhsip does not care about ses, and ses-with-rook just works without any extra config from our side.. I would say lets drop it...later because this is green.